File: Modul 3/Python/PRAK304-2310817310010-AkmallullailSyaban.py
print("Input")
nilai = int(input())

if nilai == 0:
    print("Output")
    print("Nol")
elif 1 <= nilai <= 9:
    print("Output")
    print("Satuan")
elif 10 <= nilai <= 19:
    print("Output")
    print("Belasan")
elif 20 <= nilai <= 99:
    print("Output")
    print("Puluhan")
else:
    print("Output")
    print("Anda Menginput Melebihi Limit")

File: Modul 3/C/PRAK304-2310817310010-AkmallullailSyaban.c
#include <stdio.h>

int main (){
    int input;
    printf("Input\n");
    scanf("%d",&input);
    if (input == 0 ){
        printf("Output\n");
        printf("Nol\n");
    }
    else if (1 <= input && input < 10){
        printf("Output\n");
        printf("Satuan\n");
    }
    else if (10 <= input && input <= 19){
        printf("Output\n");
        printf("Belasan\n");
    }
    else if (20 <= input && input <= 99){
        printf("Output\n");
        printf("Puluhan\n");
    }
    else {
        printf("Output\n");
        printf("Anda Menginput Melebihi Limit Bilangan\n");
    }
    return 0;
}